.evergreen/install-dependencies.sh

```diff
@@ -55,5 +55,7 @@ registry=https://registry.npmjs.org
 EOT
 
 # install node dependencies
-npm install # npm prepare runs after install and will compile the library
+# npm prepare runs after install and will compile the library
+# TODO(NODE-3555): rollup dependencies for node polyfills have broken peerDeps. We can remove this flag once we've removed them.
+npm install --legacy-peer-deps
```

Comment:
We're going to be removing the rollup plugins that are causing peer dep issues in the next ticket, so this is a temp patch to get install to keep working.